packages/terra-pills/NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright 2021-present Cerner Innovation, Inc.
```

Comment:
nit: to keep it consistent with the other file:
```suggestion
Copyright 2021 - present Cerner Innovation, Inc.
```

Reply:
Fixed 51a9979